app/bundles/ApiBundle/Config/config.php

```diff
@@ -92,10 +92,8 @@
         ],
         'models' => [
             'mautic.api.model.client' => [
-                'class'     => 'Mautic\ApiBundle\Model\ClientModel',
-                'arguments' => [
-                    'request_stack',
-                ],
+                'class'    => 'Mautic\ApiBundle\Model\ClientModel',
+                'autowire' => true,
             ],
```

Comment:
Can this and other models in this PR be moved to the services.php file instead? Something like https://github.com/mautic/mautic/blob/5.x/app/bundles/AssetBundle/Config/services.php#L23

Reply:
This is sooo tedious ;)

Reply:
I know. This all refactoring is :) I was just curious if it does the same thing. I don't like that we are adding some new logic in processing services in config.php files. I'd like to get rid of it in M6.

Reply:
Done.

Reply:
Wow, it looks amazing! Can you please resolve also the conflicts? I'll test it right after that and get it merged. This is not a BC break so it does not block the M5-alpha release.